site/docs/v1/tech/samlv2/zendesk.adoc

```diff
@@ -5,19 +5,103 @@ description: Setting up Zendesk to login using FusionAuth via SAML v2
+* `Response populate lambda` = Choose the previously created lambda.
+
+image::samlv2/zendesk/saml-config.png[Application SAML configuration.,width=1200,role=shadowed]
+
+Click save. Now view the application configuration and scroll down to the `SAML v2 Integration details` section.  Copy the following information:
+
+* `Login URL`
+* `Logout URL`
+
+image::samlv2/zendesk/saml-details-for-application.png[The SAML Integration Details.,width=1200,role=shadowed]
+
+Next, navigate to [breadcrumb]#Users# to edit the user you have previously set up or create a new one. Create a registration to add that user to the `Zendesk` application and give them the appropriate role. If you don't give them an `admin` or `agent` role, they'll default to a `end user` Zendesk role.
+
+Finally, go to [breadcrumb]#Settings -> Key Master# and view the `Signing Key` you created or chose. Copy the `Fingerprint (SHA-256)` value. This will be something like `FF:74:12:A5:40:67:E9:90:24:FC:95:07:FC:B7:E6:36:9B:26:75:6B:24:9D:3E:49:0A:43:4D:BC:03:00:DD:AA`.
+
+image::samlv2/zendesk/fingerprint-for-zendesk.png[The required certificate fingerprint.,width=1200,role=shadowed]
+
+=== Configure Zendesk
+
+The https://support.zendesk.com/hc/en-us/articles/203663676[general Zendesk SSO instructions] are worth reading.
+
+To configure Zendesk to use FusionAuth to manage your users, head to your application's security center, then to the `Single sign-on` section. This is a direct URL to that section: https://fusionauth-example.zendesk.com/admin/security/sso
+
+Enable SAML, then configure it.
+
+* The `SAML SSO URL` is the `Login URL` previously copied.
+* The `Remote Logout URL` is the `Logout URL` previously copied.
+* The `Certificate fingerprint` is the `Fingerprint (SHA-256)` value previously copied.
+
+image::samlv2/zendesk/zendesk-saml-config.png[The Zendesk SSO configuration screen.,width=1200,role=shadowed]
+
+Save the SAML configuration. The next step is to allow users to log in using the SAML integration. You can choose to let end users, staff, or both use single sign-on.
+
+Proceed to the `Staff members` section. Check `External authentication` and select `Single sign-on`. You should see that SAML is enabled. Save the configuration.
+
+Navigate to the `End users` section. Check `External authentication`. You should see that SAML is enabled. Save the configuration.
+
+You can also uncheck `Zendesk Authentication` in these two sections to ensure that users are managed only in FusionAuth. Admin users will be able to access their dashboared at `https://fusionauth-example.zendesk.com/access/normal` should FusionAuth be unavailable for any reason.
+
+New users are assigned the Zendesk `end user` role if they have no FusionAuth role. By default, users with the `user` role are sent to the Zendesk Help Center, which is not enabled by default in new Zendesk accounts. If this is the case, you'll get an error message.
+
+=== Log in
```

Comment:
We could add a "troubleshooting" section as well to reference how to bypass their FusionAuth SSO if so desired. 

Zendesk provides some bypass URLs for admins. For example, in our integration, if I want to bypass the SSO, I can go to `https://fusionauth.zendesk.com/access/normal`. (`/access/normal`)

Reply:
Great idea, broke out a troubleshooting section and added issues I ran into.